tools/system_libs.py

```diff
@@ -616,9 +607,21 @@ class Dummy(object):
 
   libs_to_link = []
   already_included = set()
-
   system_libs_map = {l.shortname: l for l in system_libs}
 
+  # Setting this in the environment will avoid checking dependencies and make building big projects a little faster
```

Comment:
This means that it forces it to build all stdlibs on demand, rather than just those needed by this compile?

Reply:
Yes, exactly.